x/masterchef/keeper/abci.go

```diff
@@ -30,12 +30,8 @@ func (k Keeper) EndBlocker(ctx sdk.Context) error {
 }
 
 func (k Keeper) GetPoolTVL(ctx sdk.Context, poolId uint64) math.LegacyDec {
-	if poolId == stabletypes.PoolId {
-		baseCurrency, found := k.assetProfileKeeper.GetUsdcDenom(ctx)
-		if !found {
-			return math.LegacyZeroDec()
-		}
-		return k.stableKeeper.TVL(ctx, k.oracleKeeper, baseCurrency)
+	if poolId >= stabletypes.PoolId {
```

Comment:
Will all the pool ids in stablestake will greater than int16 value? We are adding amm pool right?

Reply:
yes, all stablestake will be greater than int16 as USDC pool starts from max_uint16 and in stablestake we increment from there